### PR TITLE
PS-7655 Document 'audit_log_timestamps' (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Changes to be committed:
	modified:   source/management/audit_log_plugin.rst

### CODE REVIEW

File: doc/source/management/audit_log_plugin.rst [hunk 1]
```diff
@@ -680,6 +680,22 @@ This variable is used to specify the ``priority`` value for syslog. This
...
+
+   :cli: Yes
+   :scope: Global
+   :dyn: No
+   :vartype: Numeric
+   :default: UTC
+
+This variable controls the timestamp time zone added to events in the audit log. 
+
+The available vales are the following:
```
Comment (alina-derkach-oaza) [inline doc/source/management/audit_log_plugin.rst]: values

**Comment by oleksandr-kachan** [comment]
@georgelorchpercona, looks like corresponding PR is still opened https://github.com/percona/percona-server/pull/4231. Am I missing something?

File: doc/source/management/audit_log_plugin.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/management/audit_log_plugin.rst]: Fixed.

**Comment by george-lorch** [comment]
yes, it has been determined that the PR in question can not be utilized as it changes format of audit log record without an option too enable/disable.  The code change to the call that enables this cna not be controlled in this way, so we will likely not take in this PR at all.

**Pull request closed without merging**